test/python/WMComponent_t/AgentStatusWatcher_t/AgentStatusWatcher_t.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+"""
+AgentStatusWatcher test
+"""
+
+from __future__ import print_function
+import threading
+import unittest
+from WMQuality.TestInitCouchApp import TestInitCouchApp as TestInit
+
+from WMComponent.AgentStatusWatcher.AgentStatusPoller import AgentStatusPoller
+
+
+class AgentStatusWatcher_t(unittest.TestCase):
+    """
+    TestCase for WorkQueueManagerTest module
+    """
+
+    def setUp(self):
+        """
+        setup for test.
+        """
+        self.testInit = TestInit(__file__)
+        self.testInit.setLogging()
+        self.testInit.setDatabaseConnection()
+        #self.testInit.setSchema(customModules=["WMCore.WMBS"],
+        #                        useDefault=False)
+        #self.testInit.setSchema(customModules=["WMComponent.DBS3Buffer"],
+        #                        useDefault=False)
+        self.reqmonDBName = "wmstat_t"
+        self.localDBName = "wmstat_t_local"
+        self.testInit.setupCouch(self.reqmonDBName, "WMStats")
+        self.testInit.setupCouch(self.localDBName, "WMStats")
```

Comment:
Is it WMStats or WMStatsAgent for the local agent? Maybe @ticoann  knows it better

Reply:
Changed